src/contradictory_claims/claims/utils.py

```diff
@@ -0,0 +1,17 @@
+"""This file contains the utilities needed to load the claim extraction module."""
+
+import json
+import os
+
+MODEL_PATH = r"https://storage.googleapis.com/contradictory_claims_model_weights/model_crf.tar.gz"
+WEIGHT_PATH = r"https://storage.googleapis.com/contradictory_claims_model_weights/model_crf_tf.th"
```

Comment:
Is the utils.py the best place for these paths? Seems slightly strange but I'm not sure what good conventions are.